Move of AbstractModelElementInstanceTest was an API change (according to clirr plugin)

**Describe the bug**

The Maven clirr plugin detects an API incompatibility in the module `Operaton - XML Model API 1.1.0-SNAPSHOT `:

```
20:03:48,773 [INFO] --- clirr:2.8:check-no-fork (all) @ operaton-xml-model ---
20:03:48,835 [INFO] Comparing to version: 1.0.0-beta-4
20:03:48,949 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest wurde entfernt
20:03:48,949 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$AttributeAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$AttributeAssumption wurde entfernt
20:03:48,949 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$ChildElementAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$ChildElementAssumption wurde entfernt
20:03:48,949 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$TypeAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$TypeAssumption wurde entfernt
20:03:48,969 [INFO] Durchgeführt mit 4 Fehlern, 0 Warnungen und 0 anderen Änderungen.
20:03:48,969 [INFO] 
20:03:48,969 [INFO] --- clirr:2.8:check-no-fork (restrictive) @ operaton-xml-model ---
20:03:48,970 [INFO] Comparing to version: 1.0.0-beta-4
20:03:49,012 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest wurde entfernt
20:03:49,012 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$AttributeAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$AttributeAssumption wurde entfernt
20:03:49,012 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$ChildElementAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$ChildElementAssumption wurde entfernt
20:03:49,012 [ERROR] 8001: org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$TypeAssumption: Klasse org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$TypeAssumption wurde entfernt
```

**To Reproduce**

Run `mvn clean install -Pcheck-api-compatibility`

**Expected behavior**

The test succeeds

**Additional context**
See this action failure from a recent pull request:
https://github.com/operaton/operaton/actions/runs/19342140131/job/55333279618?pr=1563

--- a/model-api/xml-model/pom.xml
+++ b/model-api/xml-model/pom.xml
@@ -71,6 +71,7 @@
             <configuration>
               <excludes>
                 <exclude>org/operaton/bpm/model/xml/impl/**</exclude>
+                <exclude>org/operaton/bpm/model/xml/test/AbstractModelElementInstanceTest</exclude>
               </excludes>
             </configuration>
             <executions>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -13,7 +13,7 @@
     This pom is the parent of all Operaton projects
   </description>
   <properties>
-    <operaton.version.old>1.0.0-beta-4</operaton.version.old>
+    <operaton.version.old>1.0.0</operaton.version.old>
     <version.resteasy>3.15.6.Final</version.resteasy>
     <!-- must be in synch with transitive version from org.springframework.boot:spring-boot-starter-jersey -->
     <version.jersey3>3.1.11</version.jersey3>

